pyproject.toml

```diff
@@ -101,6 +101,7 @@ vtk = [
 # Dependencies needed to run the full test suite
 ci = [
   "ipympl",  # needed for notebook testing
+  "irksome @ git+https://github.com/firedrakeproject/Irksome.git",
```

Comment:
Is this the right way of installing a firedrake package? What are we doing for gusto?

Reply:
Gusto etc are different because we don't need them to pass our own test suite.